packages/fractal-page-object/src/-private/types.ts

```diff
@@ -9,3 +9,10 @@ export const CLONE_WITH_INDEX = Symbol('withIndex');
 export type PageObjectConstructor<T extends PageObject> = new (
   ...args: ConstructorParameters<typeof PageObject>
 ) => T | PageObject;
+
+/**
+ * Helper type for array prototype stubbing
```

Comment:
Let's update this comment since it's no longer just for array prototype stubbing -- I think `Helper type for a {@link PageObject}s and subclasses that are known to match an element`

Reply:
updated